content_scripts/link_hints.coffee

```diff
@@ -125,6 +125,22 @@ LinkHints =
     marker
 
   #
+  # Find children elements that have delegated onclick event listener assigned,
+  # and mark them with `vimium-has-delegated-onclick-listener` attribute.
+  # This will make sure those elements are discovered during `getVisibleClickableElements` call.
+  #
+  markChildrenThatHaveDelegatedOnClickListener: (element) ->
```

Comment:
Can we change this to `markElementsThatHaveDelegatedOnClickListener` and iterate over `document.querySelectorAll("*[vimium-has-delegated-onclick-listener")` instead?

Making DOM modifications in `getVisibleClickable` means that we always have to iterate over the DOM from start to finish, even if we just want to query a subset of the DOM, or query it in a different order.